sickbeard/webserve.py

```diff
@@ -3162,7 +3172,7 @@ def manageTorrents(self):
             else:
                 t.webui_url = re.sub('localhost', sickbeard.LOCALHOST_IP, sickbeard.TORRENT_HOST)
         else:
-            t.webui_url = sickbeard.TORRENT_HOST
+            t.webui_url = sickbeard.TORRENT_HOST + 'web/'
```

Comment:
@CoRpO why did you change this? 
this will break transmission.
I think you should add another conditional clause.